_summaries/maas2013rectifier_1.md

```diff
@@ -3,17 +3,45 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: kelseyball
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# Core idea: Rectifier DNNs > Sigmoidal DNNs in Speech Recognition
```

Comment:
Not completely necessary here but I think it might be nice to define the DNN acronym as the paper does before it starts to use it.